ExamenProeveUnity/Assets/Scripts/Runtime/UserInterfaces/HappinessMeterRenderer.cs

```diff
@@ -0,0 +1,25 @@
+using Runtime.Enums;
+using Runtime.Managers;
+using UnityEngine;
+
+public class HappinessMeterRenderer : MonoBehaviour
+{
+    [SerializeField] private RectTransform slider;
+    [SerializeField] private float maxHeight;
+    [SerializeField] private float minHeight;
```

Comment:
zou het niet logisch zijn geweest om parent height te pakken ipv te setten? voor nu mag het maar als je values pakt ipv moet zetten is het veel makkelijk te gebruiken